geth/api/api.go

```diff
@@ -188,3 +193,35 @@ func (api *StatusAPI) JailCall(chatID, this, args string) string {
 func (api *StatusAPI) JailBaseJS(js string) {
 	api.b.jailManager.BaseJS(js)
 }
+
+// TODO: API package this stuff
+func (api *StatusAPI) Notify(token string) string {
+	log.Info("NOTIFY LOG", "token", token)
```

Comment:
I assume this log is needed just to make sure initially the call works, right?
Should be debug(`log.Debug()`) perhaps?

Reply:
Thanks